README.md

```diff
@@ -65,7 +65,49 @@ Running the linting with auto fix:
 $ yarn lint --fix
 ```
 
-### Running the tests
+### Running Tests
+
+Due to the simplicity of this application, just checking that the application starts successfully is enough
+to be confident that it will work. You should see console output similar to the below:
+
+```
+1. Ensure wa-case-event-handler service is up. 
+    (Its dependent services also need to be up I.e. wa-workflow-api & wa-taskmanagement-service).
+
+2. Set up the following environment variables on your local wa-message-cron-service path terminal.
+    export ALLOW_CONFIG_MUTATIONS=true
+    export WA_CASE_EVENT_HANDLER_SERVICE_URL=http://localhost:{CASE_EVENT_HANDLER_SERVICE_PORT_NUMBER}
+    export JOB_NAME=FIND_PROBLEM_MESSAGES
+
+3. The s2s.secret is exclusively read from the config at runtime. For testing the application start-up locally to verify no issues:
+    edit s2s-service.ts file & set the s2s.secret to the s2s.secret value in the dev.yaml file. 
```

Comment:
Is this the one you said wasn't being set from an env var?

Reply:
Yes, that's the property. We'll raise a Jira as discussed to investigate reading the values from env variable rather than config.